composer.json

```diff
@@ -41,8 +41,8 @@
     "mimmi20/ua-device-type": "^9.0.3",
```

Comment:
Please update these 2 `mimmi20` dependencies to their latest versions (9.0.6). These new Exceptions do not exist before

Reply:
Done